documentation/staging/content/userguide/managing-domains/model-in-image/common-mounts.md

```diff
@@ -0,0 +1,444 @@
+        # "ADMIN" means the listed server will be start up to "ADMIN" mode
+        serverStartState: "RUNNING"
+        # Setup a Kubernetes node port for the administration server default channel
+        #adminService:
+        #  channels:
+        #  - channelName: default
+        #    nodePort: 30701
+    
+      # The number of managed servers to start for unlisted clusters
+      replicas: 1
+    
+      # The desired behavior for starting a specific cluster's member servers
+      clusters:
+      - clusterName: cluster-1
+        serverStartState: "RUNNING"
+        serverPod:
+          # Instructs Kubernetes scheduler to prefer nodes for new cluster members where there are not
+          # already members of the same cluster.
+          affinity:
+            podAntiAffinity:
+              preferredDuringSchedulingIgnoredDuringExecution:
+                - weight: 100
+                  podAffinityTerm:
+                    labelSelector:
+                      matchExpressions:
+                        - key: "weblogic.clusterName"
+                          operator: In
+                          values:
+                            - $(CLUSTER_NAME)
+                    topologyKey: "kubernetes.io/hostname"
+        # The number of managed servers to start for this cluster
+        replicas: 2
+    
+      # Change the restartVersion to force the introspector job to rerun
+      # and apply any new model configuration, to also force a subsequent
+      # roll of your domain's WebLogic Server pods.
+      restartVersion: '1'
+    
+      # Changes to this field cause the operator to repeat its introspection of the
+      #  WebLogic domain configuration.
+      introspectVersion: '1'
+    
+      configuration:
+    
+        # Settings for domainHomeSourceType 'FromModel'
+        model:
+          # Valid model domain types are 'WLS', 'JRF', and 'RestrictedJRF', default is 'WLS'
+          domainType: "WLS"
+          modelHome: "/common/models"
+          wdtInstallHome: "/common/weblogic-deploy"
+    
+          # Optional configmap for additional models and variable files
+          #configMap: sample-domain1-wdt-config-map
+    
+          # All 'FromModel' domains require a runtimeEncryptionSecret with a 'password' field
+          runtimeEncryptionSecret: sample-domain1-runtime-encryption-secret
+    
+        # Secrets that are referenced by model yaml macros
+        # (the model yaml in the optional configMap or in the image)
+        #secrets:
+        #- sample-domain1-datasource-secret
+  ```
+  {{% /expand %}}
+
+You can compare this domain resource YAML file with the domain resource YAML file from the initial use case (`/tmp/mii-sample/domain-resources/WLS/mii-initial-d1-WLS-v1.yaml`) to see the changes required for the common mounts option. If you created your own YAML file, then you can make the required changes for the common mounts option and create the new domain using the modified YAML file.
```

Comment:
I think it'd be helpful if right after the first sentence we added 'For example:', and then provided the diff command and its output right in a code block.